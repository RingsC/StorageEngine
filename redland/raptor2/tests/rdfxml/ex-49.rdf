<?xml version="1.0"?>
<rdf:RDF xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
         xmlns:ex="http://example.com/properties/">

  <!-- Check attributes are properly encoded when building the XML string -->

  <rdf:Description rdf:about="http://example.org/node0">
     <ex:prop1 rdf:parseType="Literal"> <foo href="http://example.org?foo=1&amp;bar=2">hmm</foo> </ex:prop1>
     <ex:prop2 rdf:parseType="Literal"> <bar a="&apos;"/> </ex:prop2>
     <ex:prop3 rdf:parseType="Literal"> <baz b="&quot;"/> </ex:prop3>
     <ex:prop4 rdf:parseType="Literal"> <baz b="&lt;&gt;"/> </ex:prop4>
  </rdf:Description>

</rdf:RDF>
