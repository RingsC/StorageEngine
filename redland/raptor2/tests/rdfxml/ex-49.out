<http://example.org/node0> <http://example.com/properties/prop1> " <foo href=\"http://example.org?foo=1&amp;bar=2\">hmm</foo> "^^<http://www.w3.org/1999/02/22-rdf-syntax-ns#XMLLiteral> .
<http://example.org/node0> <http://example.com/properties/prop2> " <bar a=\"'\"></bar> "^^<http://www.w3.org/1999/02/22-rdf-syntax-ns#XMLLiteral> .
<http://example.org/node0> <http://example.com/properties/prop3> " <baz b=\"&quot;\"></baz> "^^<http://www.w3.org/1999/02/22-rdf-syntax-ns#XMLLiteral> .
<http://example.org/node0> <http://example.com/properties/prop4> " <baz b=\"&lt;>\"></baz> "^^<http://www.w3.org/1999/02/22-rdf-syntax-ns#XMLLiteral> .
